docs/libertyDev.md

```diff
@@ -230,10 +246,10 @@ These can also be specified as command line parameters in addition to the ones i
 
 | Attribute | Type  | Since | Description | Required |
 | --------- | ----- | ----- | ----------- | -------- |
-| container | boolean | 3.1-M1 | If set to `true`, run the server in the container specified by the `dockerfile` parameter. Setting this to `true` and using the `libertyDev` task is equivalent to using the `libertyDevc` task. The default value is `false` when the `libertyDev` task is used, and `true` when the `libertyDevc` task is used. | No |
-| dockerRunOpts | String | 3.1-M1 | Specifies options to add to the `docker run` command when using dev mode to launch your server in a container. For example, `-e key=value` is recognized by `docker run` to define an environment variable with the name `key` and value `value`. | No |
-| dockerfile | File | 3.1-M1 | Location of a Dockerfile to be used by dev mode to build the Docker image for the container that will run your Liberty server. The default location is `Dockerfile` in the project root. | No |
-| dockerBuildContext | File | 3.1.3 | The Docker build context directory to be used by dev mode for the `docker build` command.  The default location is the directory of the Dockerfile. | No |
-| dockerBuildTimeout | integer | 3.1-M3 | Maximum time to wait (in seconds) for the completion of the Docker operation to build the image. The value must be an integer greater than 0. The default value is `600` seconds. | No |
-| skipDefaultPorts | boolean | 3.1-M3 | If set to `true`, dev mode will not publish the default Docker port mappings of `9080:9080` (HTTP) and `9443:9443` (HTTPS). Use this option if you would like to specify alternative local ports to map to the exposed container ports for HTTP and HTTPS using the `dockerRunOpts` parameter. | No |
-| keepTempDockerfile | boolean | 3.1 | If set to `true`, dev mode will not delete the temporary modified copy of your Dockerfile used to build the Docker image. This file is handy in case you need to debug the process of building the Docker image. The path of the temporary Dockerfile can be seen when dev mode displays the `docker build` command. The default value is `false`.| No |
+| container | boolean | 3.1-M1 | If set to `true`, run the server in the container specified by the `containerfile` parameter. Setting this to `true` and using the `libertyDev` task is equivalent to using the `libertyDevc` task. The default value is `false` when the `libertyDev` task is used, and `true` when the `libertyDevc` task is used. | No |
+| containerRunOpts | String | 3.1-M1 | Specifies options to add to the container `run` command when using dev mode to launch your server in a container. For example, `-e key=value` is recognized by `docker run` to define an environment variable with the name `key` and value `value`. | No |
```

Comment:
The "Since" column needs to reflect the release that all these attributes that start with `container` are added...which will be 3.7. We will still honor the old attributes right? Maybe add to the description of the new ones to say this replaces the now deprecated attribute? "This attribute replaces the deprecated xyz attribute."